src/screens/createWallet/VerifyMnemonicScreen.tsx

```diff
@@ -276,11 +359,8 @@ function DroppableBox({
   return (
     <Box
       ref={dropRef}
-      borderRadius="md"
-      borderWidth={1}
-      borderColor="gray.200"
-      bg={isOver ? 'blackAlpha.700' : 'blackAlpha.50'}
-      p={hasWordInside ? 0 : 2}
+      bg={hasWordInside ? '#0B221C' : 'brand.darkGreen'}
```

Comment:
Btw, now slot does not react when you drag a word over it...
Please bring back the usage of `isOver` and ensure that the color is differ from bg...